setup.py

```diff
@@ -0,0 +1,3 @@
+from distutils.core import setup
+
+setup(name='pyndustric', version='1.0.0', packages=['pyndustric'])
```

Comment:
We should be loading the `__version__` value from the root:

https://github.com/Lonami/pyndustric/blob/5c5e4228bbe60d9313c17c1c83b224183d77501b/pyndustric/__init__.py#L6

Having two values makes it error-prone since they can easily go out of sync.

I think the current solution for this is unfortunately opening the file and finding the version.

This old answer (2010) says to import the package itself and then access the variable, but in a 2017 edit says it's not recommended because that's prone to fail too: [How can I get the version defined in setup.py (setuptools) in my package?](https://stackoverflow.com/a/2058872/).

I think I'll merge as-is and worry about that later.

```suggestion
setup(name='pyndustric', version='0.1', packages=['pyndustric'])
```